addons/api/addon/generated/models/host-catalog.js

```diff
@@ -66,7 +66,6 @@ export default class GeneratedHostCatalogModel extends BaseModel {
   // AWS specific
 
   @attr('string', {
-    for: 'plugin',
```

Comment:
removed this as this was confusing, moved this to serializer to handle worker_filter, lmk if that makes sense